WanderWise-Inc/app
# Test/e2e like

# E2E test: Like Logic:
## Test flow:
- Sign In
- Like an Itinerary in the OverviewScreen
- Go to LikedScreen
- Check that the Itinerary is liked
- Unlike the Itinerary
- Go back to OverviewScreen
- Go back to LikedScreen
- Check if there is no liked Itinerary

Special thanks to @ethangraham2001 who fixed a bug with protobuf in commit: 
- [Fixed weird dependency issue with protobufs that was breaking](https://github.com/WanderWise-Inc/app/pull/277/commits/c7343f744e38b6e5d206a4272edf5c94b161f33b)
